libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -120,6 +122,13 @@ class BaseChatModel(BaseLanguageModel[BaseMessage], ABC):
     callback_manager: Optional[BaseCallbackManager] = Field(default=None, exclude=True)
     """[DEPRECATED] Callback manager to add to the run trace."""
 
+    exceptions_to_handle: Tuple[Type[BaseException], ...] = ()
```

Comment:
do we need this? could just always yield remove messages on error (assuming we always raise error anyways)

Reply:
yea if we always raise we don't need!